pkg/gatewayserver/io/udp/udp.go

```diff
@@ -57,10 +57,10 @@ type Config struct {
 var DefaultConfig = Config{
 	PacketHandlers:      10,
 	PacketBuffer:        50,
-	DownlinkPathExpires: 30 * time.Second,
-	ConnectionExpires:   5 * time.Minute,
+	DownlinkPathExpires: 15 * time.Second, // Expire downlink after missing typically 3 PULL_DATA messages.
+	ConnectionExpires:   40 * time.Second, // Expire connection after missing typically 1 status message.
```

Comment:
The chances of missing a single message are pretty high. This may have unintended results. Perhaps we should just make it the same as v2 (1 minute)?